### PR TITLE
[#87] Add possibility to add classes on h element…

### PR DESCRIPTION
**Pull request by andreasknoepfle**
…for ui_section_title

Also on the side, this makes sure all the classes bitstyles_phoenix sets run through classnames()

https://github.com/bitcrowd/bitstyles_phoenix/issues/87

### CODE REVIEW

File: lib/bitstyles_phoenix/component/breadcrumbs.ex [hunk 1]
```diff
@@ -182,7 +182,7 @@ defmodule BitstylesPhoenix.Component.Breadcrumbs do
               {assigns[:item] && assigns_to_attributes(item, [:class]) || []}
             >
               <%= if assigns[:item], do: render_slot(item), else: item %>
-              <.ui_icon name="caret-right" size="m" class="u-fg-gray-30 u-margin-xs-left" {@icon_extra} />
+              <.ui_icon name="caret-right" size="m" class={classnames("u-fg-gray-30 u-margin-xs-left")} {@icon_extra} />
```
Comment (hannahbit) [inline lib/bitstyles_phoenix/component/breadcrumbs.ex:185]: Oh! Great ✨

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 ### Changed
+- Add `heading_class` option to `ui_heading_title` to set classes on the heading
```
Comment (hannahbit) [inline CHANGELOG.md:5]: It is `ui_section_title`, right?

```suggestion
- Add `heading_class` option to `ui_section_title` to set classes on the heading
```

**Review by hannahbit — COMMENTED** [review]
Great changes! 💜 

I think it would be great if we would update the `@doc` of `ui_section_title` and mention the new possible atttribute.

**Review by planktonic — COMMENTED** [review]
Apart from what Hannah already said, looks good to me 🎉

**Pull request merged**